packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx

```diff
@@ -0,0 +1,306 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
Can you add some information on this new example?

Reply:
Added 35c8ed6